lms/templates/problem.html

```diff
@@ -88,7 +88,7 @@ <h3 class="hd hd-3 problem-header" id="${ short_id }-problem-title" aria-describ
       <div class="submission-feedback ${'cta-enabled' if submit_disabled_cta else ''}" id="submission_feedback_${short_id}">
         ## When attempts are not 0, the CTA above will contain a message about the number of used attempts
         % if attempts_allowed and (not submit_disabled_cta or attempts_used == 0):
-          ${ngettext("You have used {num_used} of {num_total} attempt", "You have used {num_used} of {num_total} attempts", attempts_allowed).format(num_used=attempts_used, num_total=attempts_allowed)}
+          ${ngettext("You have used {num_used} of {num_total} attempt. Grading strategy: {grading_strategy}", "You have used {num_used} of {num_total} attempts. Grading strategy: {grading_strategy}", attempts_allowed).format(num_used=attempts_used, num_total=attempts_allowed, grading_strategy=grading_strategy)}
```

Comment:
Should we show the grading strategy for the learner as is? Should we add a description? I'm not sure what's the best way of explaining what each of them does.